src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx

```diff
@@ -531,215 +515,24 @@ const UnitsTab: FC<UnitsTabProps> = ({ data, examboardSlug }) => {
-                                  data-testid="unit-modal-button"
-                                  variant={isHighlighted ? "brush" : "minimal"}
-                                  background={
-                                    isHighlighted ? "black" : undefined
-                                  }
-                                  label="Unit info"
-                                  onClick={() => {
-                                    handleOpenModal();
-                                    setUnitOptionsAvailable(unitOptions);
-                                    setUnitData({ ...unit });
-                                    setCurrentUnitLessons(unit.lessons ?? []);
-                                  }}
-                                  ref={modalButtonRef}
-                                />
-                              </OakFlex>
-                            </Card>
-                          );
-                        })}
-                      <UnitsTabSidebar
-                        displayModal={displayModal}
-                        onClose={handleCloseModal}
-                        lessons={currentUnitLessons}
-                        programmeSlug={createProgrammeSlug(
-                          unitData,
-                          examboardSlug,
-                        )}
-                        unitOptionsAvailable={unitOptionsAvailable}
-                        unitSlug={unitData?.slug}
-                        unitVariantID={unitVariantID}
-                      >
-                        <UnitModal
-                          setCurrentUnitLessons={setCurrentUnitLessons}
-                          setUnitVariantID={setUnitVariantID}
-                          unitData={unitData}
-                          displayModal={displayModal}
-                          setUnitOptionsAvailable={setUnitOptionsAvailable}
-                          unitOptionsAvailable={unitOptionsAvailable}
-                          isHighlighted={
-                            unitData ? isHighlightedUnit(unitData) : false
-                          }
-                        />
-                      </UnitsTabSidebar>
-                    </OakFlex>
-                  </Box>
-                );
-              })}
-          </OakGridArea>
+          <CurriculumVisualiser
+            unitData={unitData}
+            yearSelection={yearSelection}
+            selectedYear={selectedYear}
+            examboardSlug={examboardSlug}
+            yearData={yearData}
+            handleSelectDomain={handleSelectDomain}
+            handleSelectSubject={handleSelectSubject}
+            handleSelectTier={handleSelectTier}
+            duplicateUnitSlugs={duplicateUnitSlugs}
+            mobileHeaderScrollOffset={mobileHeaderScrollOffset}
+            setUnitData={setUnitData}
+            selectedThread={selectedThread}
+            setVisibleMobileYearRefID={setVisibleMobileYearRefID}
+          />
```

Comment:
I think this was a good move. Maybe a separate PR as you mentioned in slack :). I can see that a few of my comments were really reviewing the structure of this and you are just moving it (as it was). We can create a separate PR for the smaller components I mentioned earlier `Year` & `Unit`. Up to you if you want to move modal in this PR also.